document-translator/script.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import './pdf.min.mjs';
+
+const {pdfjsLib} = globalThis;
+pdfjsLib.GlobalWorkerOptions.workerSrc = './pdf.worker.mjs';
+
+const [openButton, exampleButton, languageButton, translateButton] = Array.from(document.querySelectorAll('button'));
+const output = document.querySelector('output');
+const detectedLanguageParagraph = document.querySelector('.detected-language');
+const translationParagraph = document.querySelector('.translation');
+const img = document.querySelector('img');
+
+let useExample = false;
+let detectedLanguage = false;
```

Comment:
Should it be an `undefined` value instead of `false` as it's supposed to be valid language value eventually?

Reply:
`undefined` it is.